lib/main.dart

```diff
@@ -208,6 +208,11 @@ class _TenTenOneState extends State<TenTenOneApp> {
           .then((value) => FLog.info(text: "ldk node stopped."))
           .catchError((error) => FLog.error(text: "ldk stopped with an error", exception: error));
 
+      // connect to the maker, this will not return unless there was an error.
+      api.connect().then((value) => FLog.error(text: "Lost connection to the maker")).catchError(
+          (error) =>
+              FLog.error(text: "Lost connection to the maker with an error", exception: error));
```

Comment:
❓ Just for my understanding: The comment says that this shouldn't return unless there is an error, but there is more code that needs to run after this. Does this work because we are not `await`ing this future?

Reply:
yes.. with`then` and `catchError` we register callback functions in case the function returns.. we do the same with runLdk.